src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/KotlinJvmTaskExecutor.kt

```diff
@@ -210,6 +210,7 @@ class KotlinJvmTaskExecutor @Inject internal constructor(
         expandWithSources(
             File(directories.generatedSources).walkTopDown()
                 .filter { it.isFile }
```

Comment:
combine `it.isFile && IS_JVM_SOURCE_FILE.test(it.name)` for a single pass?

Reply:
Done.